week7/young-hee/my-app/src/pages/Community.jsx

```diff
@@ -0,0 +1,22 @@
+import React from'react';
+import styled from 'styled-components';
+
+const CommunityStyle = styled.div`
+  margin: 0 auto;
+  margin-top : 300px;
+  margin-left : 300px;
```

Comment:
이런 경우는 한줄로 줄여볼 수 있지 않을까요? (+ 그런데 궁금한 점이, 가운데 정렬을 위해 `margin: 0 auto` 하신것 아닐까요? 그렇다면 margin-left 속성이 필요한 이유가 무엇일까요?)
```suggestion
  margin: 300px auto 0 300px;
```